www/js/app.js

```diff
@@ -328,56 +328,56 @@ $('#prefix').on('keydown', function (e) {
     }
 });
 // Search for titles as user types characters
-$('#prefix').on('keyup', function (e) {
+document.getElementById('prefix').addEventListener('keyup', function (e) {
     if (selectedArchive !== null && selectedArchive.isReady()) {
         // Prevent processing by keyup event if we already handled the keypress in keydown event
         if (keyPressHandled) { keyPressHandled = false; } else { onKeyUpPrefix(e); }
     }
 });
 // Restore the search results if user goes back into prefix field
-$('#prefix').on('focus', function () {
+document.getElementById('prefix').addEventListener('focus', function () {
     if (document.getElementById('prefix').value !== '') { document.getElementById('articleListWithHeader').style.display = ''; }
 });
 // Hide the search results if user moves out of prefix field
-$('#prefix').on('blur', function () {
+document.getElementById('prefix').addEventListener('blur', function () {
     if (!searchArticlesFocused) {
         appstate.search.status = 'cancelled';
         document.getElementById('searchingArticles').style.display = 'none';
         document.getElementById('articleListWithHeader').style.display = 'none';
     }
 });
-$('#btnRandomArticle').on('click', function () {
+document.getElementById('btnRandomArticle').addEventListener('click', function () {
     document.getElementById('prefix').value = '';
     goToRandomArticle();
     document.getElementById('welcomeText').style.display = 'none';
     document.getElementById('articleListWithHeader').style.display = 'none';
     $('.navbar-collapse').collapse('hide');
 });
 
-$('#btnRescanDeviceStorage').on('click', function () {
+document.getElementById('btnRescanDeviceStorage').addEventListener('click', function () {
     searchForArchivesInStorage();
 });
 // Bottom bar :
-$('#btnBack').on('click', function () {
+document.getElementById('btnBack').addEventListener('click', function () {
     history.back();
     return false;
 });
 $('#btnForward').on('click', function () {
     history.forward();
     return false;
```

Comment:
Check all the `return false` statements below and either convert or remove as needed.